packages/core/src/utils/logger.ts

```diff
@@ -0,0 +1,5 @@
+import { client } from '../client'
+
+export function logWarning(message: string) {
+  client?.config.logger?.warn?.(message)
+}
```

Comment:
Let's use `getClient` and call it `logWarn`?

```suggestion
import { getClient } from '../client'

export function logWarn(message: string) {
  getClient()?.config.logger?.warn?.(message)
}
```

Reply:
sg!